solace-spring-cloud-starters/solace-spring-cloud-stream-starter/README.adoc

```diff
@@ -989,15 +973,15 @@ A simple error handling strategy in which failed messages are redelivered from t
 
 [IMPORTANT]
 ====
-The Solace API used in this binder implementation does not support individual message redelivery.
+The internal implementation of redelivery has changed from Solace Binder v5.0.0.
+Previously, redelivery was initiated by rebinding consumer flows; however, as of v5.0.0 and later, the Solace API now leverages the Solace broker's native NACK (Negative Acknowledgement) capabilities.
 
 Here is what happens under the hood when this is triggered:
 
-1. The Solace flow receiver is stopped.
-2. Wait until all unacknowledged messages have been acknowledged with a maximum timeout of `flowPreRebindWaitTimeout`. If timed out, the remaining unacknowledged messages will be stale and redelivered from the broker.
-3. Rebind the flow.
+1. Say the current message is marked for 'REQUEUE'. Any subsequent messages that are currently spooled on the client side, despite having been acknowledged `ACCEPTed` by binder, the Solace broker will discard their ACK.
+2. The Solace Broker will redeliver all messages starting with the one tagged as 'REQUEUE', if the message's max redelivery count is not exceeded.
```

Comment:
Same question as the other PS+ broker question: Should we be this specific about what the broker does when handling settlements?

IMO I think it might be better to have a short description about this and link to the official docs that describes these details (assuming it exists).